Project 1 (9.12)/stoper/stoper.js

```diff
@@ -1,5 +1,50 @@
+const startBtn = document.querySelector("#button-start");
+const stopBtn = document.querySelector("#button-stop");
+const resetBtn = document.querySelector("#button-reset");
+const secondsBtn = document.querySelector("#seconds");
+const tensBtn = document.querySelector("#tens");
 function Stoper() {
-    
-};
-
-Stoper();
\ No newline at end of file
+  let tens = 0;
+  let seconds = 0;
+  let startCounting;
+  let stopCounting;
+  let start = () => {
+    clearInterval(startCounting);
+    startCounting = setInterval(start, 10);
+    tens = tens + 1;
+    tensBtn.innerHTML = "0" + tens;
+    secondsBtn.innerHTML = "0" + seconds;
+    if (tens >= 10) {
+      tensBtn.innerHTML = tens;
+    }
+    if (seconds >= 10) {
+      secondsBtn.innerHTML = seconds;
+    }
+    if (tens === 99) {
+      tens = -1;
```

Comment:
a czemu -1?

Reply:
bo chcielismy,  zeby po 99 nie bylo 100 tylko , zeby sie zerowalo. A gdy zostawilismy 0 to po paru cyklach było minimalnie rozstrojone